tests/cps-files/lib/pkgconfig/pc-comments.pc

```diff
@@ -0,0 +1,5 @@
+# Comment
```

Comment:
I had a bug in my elsewhere hand rolled pc parser for:

```
# Name: Something
```

And another for:

```
Requires: foo bar # baz
```

Depending on how comments are lexed, they might be moot to add tests for, though.

Reply:
```
Requires: foo bar # baz
```
This case is covered by the next line.
```
# Name: Something
```
Let me add something for this